bin/butane.js

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env node
 
-var argv = require('minimist')(process.argv.slice(2));
-var fireRules = require('../dist/');
-
-var input = argv._[0];
-var output = argv._[1];
-
-fireRules.convert(input, output);
+require('require-relative-main')('./cli', __dirname)
```

Comment:
If you insist, but let me make the case for dependencies first. The dependency is a generic solution. It'll work even if `main` were to change as long as the structure of the files remains the same. If you hard code the path, everything breaks. Say you have main be `./lib` and then change it to `./dist` on prepublish and back to `./lib` on postpublish. That's actually the correct way to do transpilation, so your `main` always points to fresh code. Now you can't hardcode dist all over the place. 

A well tested, well documented dependency prevents someone from ever thinking about this again.